cmd/containerd/command/main.go

```diff
@@ -243,6 +257,9 @@ can be used and modified as necessary as a custom configuration.`
 func serve(ctx gocontext.Context, l net.Listener, serveFunc func(net.Listener) error) {
 	path := l.Addr().String()
 	log.G(ctx).WithField("address", path).Info("serving...")
+	serveSpan, ctx := tracing.StartSpan(ctx, l.Addr().String())
```

Comment:
should this happen if tracing is not enabled in the config?

Reply:
same as above answer